cli/cmd/component.go

```diff
@@ -349,7 +350,16 @@ func componentsToTable() [][]string {
 	return out
 }
 
-func runComponentsInstall(_ *cobra.Command, args []string) (err error) {
+func runComponentsInstall(cmd *cobra.Command, args []string) (err error) {
+	var (
+		component_name string
```

Comment:
Nit- Not the standard way to name variables
```suggestion
		componentName string
```